src/installation/guides/chroot.md

```diff
@@ -0,0 +1,232 @@
+
+Use xbps to boot strap the distribution by installing the `base-system`
+metapackage.
+
+```
+# XBPS_ARCH=$ARCH xbps-install -S -r /mnt -R "$REPO" base-system
+```
+
+## Configuration
+
+### Entering the Chroot
+
+Mount the pseudo-filesystems needed for a chroot,
+
+```
+# mount -t proc /proc /mnt/proc
+# mount -t sysfs /sys /mnt/sys
+# mount -o bind /dev /mnt/dev
+# mount -t devpts pts /mnt/dev/pts
+```
+
+Copy the DNS configuration into the new root so that xbps can still download new
+packages inside the chroot,
+
+```
+# cp /etc/resolv.conf /mnt/etc/
+```
+
+Chroot into the new installation
+
+```
+# PS1='(chroot) # ' chroot /mnt/ /bin/bash
+```
+
+Write the hostname to `/mnt/etc/hostname`. Go through the options in
+[`/mnt/etc/rc.conf`](../../config/rc-files.md#rcconf). If installing a glibc
+distribution, edit `/mnt/etc/default/libc-locales`, uncommenting desired
+[locales](../../config/locales.md).
+
+[nvi(1)](https://man.voidlinux.org/nvi.1) is available in the chroot, but you
+may wish to install your preferred text editor at this time.
+
+For glibc builds, generate locale files with
+
+```
+(chroot) # xbps-reconfigure -f glibc-locales
+```
+
+[Configure at least one super user account](../../config/users-and-groups.md).
+Other user accounts can be configured later, but there should either be a root
+password, or a new user account with [sudo(8)](https://man.voidlinux.org/sudo.8)
+privileges.
+
+To set a root password, run
+
+```
+(chroot) # passwd
+```
+
+Configure [fstab(5)](https://man.voidlinux.org/fstab.5). This file can be
+automatically generated from currently mounted filesystems by copying the file
+`/proc/mounts`.
+
+```
+(chroot) # cp /proc/mounts /etc/fstab
+```
+
+Remove lines in `/etc/fstab` that refer to `proc`, `sys`, `devtmpfs` and `pts`.
+
+Replace references to `/dev/sdXX` with their respective UUID, which can be found
+by running `lsblk -f` (see [lsblk(8)](https://man.voidlinux.org/lsblk.8)).
+
+Change the last zero of the entry for `/` to `1`, and the last zero of every
+other line to `2`. These values configure the behaviour of
+[fsck(8)](https://man.voidlinux.org/fsck.8).
+
+For example, the partition scheme used throughout previous examples yields the
+following fstab:
+
+```
+/dev/sda1       /boot/EFI   vfat    rw,relatime,[...]       0 0
+/dev/sda2       /           ext4    rw,relatime             0 0
+```
+
+The fstab is modified thus with information from `lsblk -f`:
+
+```
+UUID=6914[...]  /boot/EFI   vfat    rw,relatime,[...]       0 2
+UUID=dc1b[...]  /           ext4    rw,relatime             0 1
+```
+
+> Note: The output of `/proc/mounts` will have a single space between each
+> field. The columns aligned here for readability.
+
+Add an entry to mount `/tmp` in ram:
+
+```
+tmpfs           /tmp        tmpfs   defaults,nosuid,nodev   0 0
+```
+
+If using swap space, add an entry for any swap partitions:
+
+```
+UUID=1cb4[...]  swap        swap    rw,noatime,discard      0 0
+```
+
+## Installing Grub
```

Comment:
I *think* it's "GRUB". The wikipedia article is all over the place with this, but their website only uses Grub when referring to Grub Legacy.

Reply:
okay, I've changed it to GRUB except where I'm referring to a void package